Containers/Deploy-.NET-Core-app-in-Windows-Containers-to-ECS/Readme.md

```diff
@@ -235,6 +250,17 @@ The container's public IP address is then listed in the *Network bindings* secti
 
 Click on the URL under the *External Link* section, and the ASP.NET Core site will load in your browser.
 
-![App](media/image5.png)
+![App](media/image5_updated.PNG)
 
 Congratulations, you have now deployed an ASP.NET Core application to Amazon Elastic Container Service!
+
```

Comment:
Is this a duplicate section? I can se troubleshooting above too.

Reply:
Oh yes, this is duplicate which I was initially testing at the end of the page. Removed it!